core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerProxy.scala

```diff
@@ -613,7 +612,7 @@ class FunctionPullingContainerProxy(
         logging.info(
           this,
           s"Remain running activations ${runningActivations.keySet().toString()} when received ClientClosed")
-        setTimer(RunningActivationTimeoutName, ClientClosed, runningActivationTimeout)
+        startSingleTimer(RunningActivationTimeoutName, ClientClosed, runningActivationTimeout)
```

Comment:
wondering if `startTimerAtFixedRate`​ should be used instead

Reply:
https://github.com/apache/openwhisk/pull/5123/files#r646383365